datacontract/imports/dbt_importer.py

```diff
@@ -74,27 +78,81 @@ def import_dbt_manifest(
         dc_model = Model(
             description=model_contents.description,
             tags=model_contents.tags,
-            fields=create_fields(columns=model_contents.columns, adapter_type=adapter_type),
+            fields=create_fields(
+                manifest,
+                model_unique_id=model_contents.unique_id,
+                columns=model_contents.columns,
+                adapter_type=adapter_type,
+            ),
         )
 
         data_contract_specification.models[model_contents.name] = dc_model
 
     return data_contract_specification
 
+
 def convert_data_type_by_adapter_type(data_type: str, adapter_type: str) -> str:
     if adapter_type == "bigquery":
         return map_type_from_bigquery(data_type)
     return data_type
 
 
-def create_fields(columns: dict[str, ColumnInfo], adapter_type: str) -> dict[str, Field]:
-    fields = {
-        column.name: Field(
-            description=column.description,
-            type=convert_data_type_by_adapter_type(column.data_type, adapter_type) if column.data_type else "",
-            tags=column.tags,
+def create_fields(
+    manifest: Manifest, model_unique_id: str, columns: dict[str, ColumnInfo], adapter_type: str
+) -> dict[str, Field]:
+    fields = {column.name: create_field(manifest, model_unique_id, column, adapter_type) for column in columns.values()}
+    return fields
+
+
+def get_column_tests(manifest: Manifest, model_name: str, column_name: str) -> list[dict[str, str]]:
+    column_tests = []
+    model_node = manifest.nodes.get(model_name)
+    if not model_node:
+        raise ValueError(f"Model {model_name} not found in manifest.")
+
+    model_unique_id = model_node.unique_id
+    test_ids = manifest.child_map.get(model_unique_id, [])
+
+    for test_id in test_ids:
+        test_node = manifest.nodes.get(test_id)
+        if not test_node or test_node.resource_type != "test":
+            continue
+
+        if not isinstance(test_node, GenericTestNode):
+            continue
+
+        if test_node.column_name != column_name:
+            continue
+
+        if test_node.config.where is not None:
```

Comment:
In tests for dbt that include `not_null`, you can use the `where` keyword to add conditions. When comparing the `required` and `not_null` [specifications of datacontract](https://datacontract.com/), we decided to exclude the `where` keyword from the current specification, as it is difficult to describe the `required` condition in datacontract when the `where` keyword is specified.

- https://docs.getdbt.com/reference/resource-configs/where